GPU/GPUCommon.cpp

```diff
@@ -366,11 +378,47 @@ GPUCommon::GPUCommon(GraphicsContext *gfxCtx, Draw::DrawContext *draw) :
 	gstate.Reset();
 	gstate_c.Reset();
 	gpuStats.Reset();
+
+	memset(cmdInfo_, 0, sizeof(cmdInfo_));
+
+	// Import both the global and local command tables, and check for dupes
```

Comment:
Hmm, this comment is wrong now.

-[Unknown]

Reply:
Indeed so, will fix.